packages/react-menu/config/normalize-import.js

```diff
@@ -0,0 +1,76 @@
+// @ts-check
+
+/// <reference lib="es2019.array" />
+
+const fs = require('fs');
+const path = require('path');
+const { default: chalk } = require('chalk');
+const yargsParser = require('yargs-parser');
+
+/**
+ * @type {Console['log']}
+ */
+// eslint-disable-next-line no-console
+const log = console.log.bind(console);
+
+main();
+
+/**
+ * Normalize cross package type imports in generated ambient types
+ *
+ * NOTE:
+ * This is a temporary workaround to TypeScript bug that occurs when path aliases are used
+ * For more info see: https://github.com/microsoft/TypeScript/issues/42349
+ */
+function main() {
+  /**
+   * @type {import('yargs').Arguments<{output?:string}>}
+   */
+  const args = yargsParser(process.argv.slice(2));
+
+  if (!args.output) {
+    throw new Error(`--output is missing`);
+  }
+
+  log(chalk.blue('Normalize ambient declarations...'));
+
+  const files = getAllFiles(path.resolve(process.cwd(), args.output));
+  normalizeImports(files);
+
+  log(chalk.blue('Normalize ambient declarations: ✅'));
+}
+
+/**
+ *
+ * @param {string[]} files
+ */
+function normalizeImports(files) {
```

Comment:
I know this is very naive, but I don't think we need to go AST way for such a simple task :)

Reply:
I am just thankful that our folder names reflec the package name... otherwise this might have been a nightmare

Reply:
Can you comment somewhere that this just assumes the above fact and just replaces relative `.` dots with the `@fluentui/scope`

Reply:
hmm is that needed? I mean that's the only issue why is this needed - local package deps, besides that TS works as expected.  

Also there are tests :)

Reply:
That's good point 👍

Reply:
It's true that the tests help with documentation, but it wouldn't hurt to mention the reason inline as well for anyone who comes across this later and isn't familiar with the whole scenario.